src/io/file_system.ts

```diff
@@ -110,39 +112,57 @@ export class NodeFileSystem implements tfc.io.IOHandler {
       };
     }
   }
-
   async load(): Promise<tfc.io.ModelArtifacts> {
-    if (Array.isArray(this.path)) {
-      throw new Error('Loading from multiple paths is not supported yet.');
-      // TODO(cais, nkreeger): Implement this. See
-      //   https://github.com/tensorflow/tfjs/issues/343
-    }
+    return Array.isArray(this.path) ? this.loadBinaryModel() :
+                                      this.loadJSONModel();
+  }
 
-    const info = await stat(this.path).catch(doesNotExistHandler('Path'));
+  protected async loadBinaryModel(): Promise<tfc.io.ModelArtifacts> {
+    const topology =
+        await stat(this.path[0]).catch(doesNotExistHandler('Path'));
+    const weightManifest =
+        await stat(this.path[1]).catch(doesNotExistHandler('Path'));
 
     // `this.path` can be either a directory or a file. If it is a file, assume
     // it is model.json file.
+    if (topology.isFile() && weightManifest.isFile()) {
+      const modelTopology = await readFile(this.path[0]);
```

Comment:
nit: I'd save these paths as local variables to the method:

```ts
protected async loadBinaryModel(): Promise<tfc.io.ModelArtifacts> {
  const topologyPath = this.path[0];
  const weightManifestPath = this.path[1];
  ...
```

Might make this easier if we change what `this.path` is down the line.